packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -15,7 +45,7 @@ export class SessionReplayPlugin implements EnrichmentPlugin {
     this.options = { ...options };
   }
 
-  async setup(config: BrowserConfig) {
+  async setup(config: BrowserConfig, client: BrowserClient) {
```

Comment:
```suggestion
  async setup(config: BrowserConfig, client?: BrowserClient) {
  
  if (!client) {
    config.loggerProvider.error('...')
    return;
  }
```

i think to be safe, we can make client optional and handle it accordingly to prevent runtime errors for versions <= 1.9.0.

Reply:
Fixed.